### PR TITLE
🧹 update readme

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,10 +66,60 @@ export GITHUB_TOKEN=<personal_access_token>
...
+| Platform            | Versions                 |
+|---------------------|--------------------------|
+| Alpine              | 3.10 - 3.16              |
+| AlmaLinux           | 8                        |
+| Amazon Linux        | 1, 2, 2022               |
+| Arch Linux          | Rolling                  |
+| CentOS              | 6, 7                     |
+| Debian              | 8, 9, 10, 11             |
+| Fedora              | 30 - 36                  |
+| OpenSuse            | Leap 15.4                |
```
Comment (tas50) [inline README.md]:
```suggestion
| openSUSE            | Leap 15.4                |
```

File: README.md [hunk 2]
```diff
@@ -63,10 +66,60 @@ export GITHUB_TOKEN=<personal_access_token>
...
+cnspec vuln vsphere user@host --ask-pass
+
+# scan Linux, Windows
+cnspec vuln local
+```
+
+| Platform            | Versions                 |
+|---------------------|--------------------------|
+| Alpine              | 3.10 - 3.16              |
+| AlmaLinux           | 8                        |
```
Comment (tas50) [inline README.md]: We do 9 as well

File: README.md [hunk 3]
```diff
@@ -63,10 +66,60 @@ export GITHUB_TOKEN=<personal_access_token>
...
+| Amazon Linux        | 1, 2, 2022               |
+| Arch Linux          | Rolling                  |
+| CentOS              | 6, 7                     |
+| Debian              | 8, 9, 10, 11             |
+| Fedora              | 30 - 36                  |
+| OpenSuse            | Leap 15.4                |
+| Oracle Linux        | 6, 7,8                   |
```
Comment (tas50) [inline README.md]:
```suggestion
| Oracle Linux        | 6, 7, 8                  |
```

File: README.md [hunk 4]
```diff
@@ -63,10 +66,60 @@ export GITHUB_TOKEN=<personal_access_token>
...
+| Photon Linux        | 2, 3 ,4                  |
```
Comment (tas50) [inline README.md]:
```suggestion
| Photon Linux        | 2, 3 , 4                 |
```

File: README.md [hunk 5]
```diff
@@ -63,10 +66,60 @@ export GITHUB_TOKEN=<personal_access_token>
...
+| Red Hat             | 6, 7, 8                  |
+| Rocky Linux         | 8                        |
+| Suse  SLES          | 12, 15                   |
```
Comment (tas50) [inline README.md]:
```suggestion
| SUSE Linux Enterprise          | 12, 15                   |
```

File: README.md [hunk 6]
```diff
@@ -63,10 +66,60 @@ export GITHUB_TOKEN=<personal_access_token>
...
+| AlmaLinux           | 8                        |
+| Amazon Linux        | 1, 2, 2022               |
+| Arch Linux          | Rolling                  |
+| CentOS              | 6, 7                     |
+| Debian              | 8, 9, 10, 11             |
+| Fedora              | 30 - 36                  |
+| OpenSuse            | Leap 15.4                |
+| Oracle Linux        | 6, 7,8                   |
+| Photon Linux        | 2, 3 ,4                  |
+| Red Hat             | 6, 7, 8                  |
```
Comment (tas50) [inline README.md]:
```suggestion
| Red Hat Enterprise Linux            | 6, 7, 8                  |
```

**Comment by chris-rock** [comment]
Thank you @tas50 for all the suggestions

**Review by tas50 — APPROVED** [review]

**Pull request merged**